src/Main.purs

```diff
@@ -223,15 +223,17 @@ succeeded :: Unit -> EffN Unit
 succeeded = const (pure unit)
 
 main :: EffN Unit
-main = runAff failed succeeded do
-  requireNodeAtLeast (version 4 0 0 Nil Nil)
-  argv <- drop 2 <$> liftEff Process.argv
-  args <- parse globals commands argv
-  case args of
-    Left (ParseError { message: err }) ->
-      handleParseError (head argv) err
-    Right args' -> do
-      runArgs args'
+main = do 
+  runAff failed succeeded do
+    requireNodeAtLeast (version 4 0 0 Nil Nil)
+    argv <- drop 2 <$> liftEff Process.argv
+    args <- parse globals commands argv
+    case args of
+      Left (ParseError { message: err }) ->
+        handleParseError (head argv) err
+      Right args' -> do
+        runArgs args'
+  pure unit
```

Comment:
Ahh, I see, thanks. I think `void x` is generally better style than `x >>= \_ -> pure unit`, since it only requires Functor (rather than Monad) and I think it is a bit more clear about its purpose, so I would consider using `void` here instead; what do you think?

Don't worry about changing that just yet though, as I haven't yet decided what to do about the Aff changes in Pulp.Watch and that might affect this.